apps/chat/README.md

```diff
@@ -112,6 +112,7 @@ AI DIAL Chat uses environment variables for configuration. All environment varia
 | `IS_IFRAME`                         |    No    | Is iFrame flag enables/disables the [DIAL Overlay](https://github.com/sr-remsha/ai-dial-chat/blob/docs-revew/libs/overlay/README.md)                                                                                                                                                                                          | `true`, `false`                                                                                  | `false`                                                                                                                            |
 | `ENABLED_FEATURES`                  |    No    | Features enabled in the AI DIAL Chat application                                                                                                                                                                                                                                                                              | Refer to [Features](../../libs/shared/src/types/features.ts) to view all the available features. |                                                                                                                                    |
 | `PUBLICATION_FILTERS`               |    No    | Defines types of publications that can be retrieved from the database                                                                                                                                                                                                                                                         | Any string. Values must be separated by a comma.                                                 | `title,role`                                                                                                                       |
+| `ADMIN_ROLE_NAME`                   |    No    | Defines the name of the administrator role                                                                                                                                                                                                                                                                                    | Any string                                                                                       | `admin`                                                                                                                            |
```

Comment:
What do you think of having array of such names? Maybe they will have multiple admin roles to differentiate different available admin possibilities